pkg/vulnsrc/nvd/nvd.go

```diff
@@ -160,3 +164,17 @@ func getCvssV3(metricsV31, metricsV30 []CvssMetricV3) (score float64, vector str
 	}
 	return
 }
+
+// getCvssV40 selects vector, score and severity from V40 metrics
+func getCvssV40(metricsV40 []CvssMetricV40) (score float64, vector string, severity types.Severity) {
+	for _, metricV40 := range metricsV40 {
+		// save only NVD metric
+		if metricV40.Source == nvdSource {
+			score = metricV40.CvssData.BaseScore
+			vector = metricV40.CvssData.VectorString
+			severity, _ = types.NewSeverity(metricV40.CvssData.BaseSeverity)
```

Comment:
It seems like the vector string is unnecessarily long with undefined metrics.
https://github.com/aquasecurity/vuln-list-nvd/blob/03b5edfbe2767b98dd949c53326399d301533486/api/2024/CVE-2024-5732.json#L24C28-L24C202

We can trim those metrics and compact the vector. github.com/pandatix/go-cvss returns a minimal vector.
https://github.com/aquasecurity/vuln-list-nvd/blob/03b5edfbe2767b98dd949c53326399d301533486/api/2024/CVE-2024-5732.json#L24C28-L24C202

Reply:
fix it
https://github.com/aquasecurity/trivy-db/pull/414/commits/303aa1827e700e5815c20520bb3116272b95bb1b